### PR TITLE
Font loading review

### PR DESCRIPTION
**Pull request by qsm-odoo**

### CODE REVIEW

**Comment by qsm-odoo** [comment]
@mart-e Could you tell me if this feels ok to you: https://github.com/odoo/odoo/pull/37293/commits/9998268a03868f7f7e1f15f9157b8efba4d87dad ?

**Comment by qsm-odoo** [comment]
@robodoo rebase-ff

**Comment by mart-e** [comment]
> Avoid name conflict for the support font

Do you get a font named exactly 'Noto' ? As the expected is something like 'Noto Bold...' you should not have conflicts. But I don't have issue to clarify the name.

Nice one the sass function :+1:

**Comment by qsm-odoo** [comment]
It was mainly to be sure it was the Odoo one I saw in my browser console while I tested so it's probably more to clarify the name than to avoid name conflict indeed. I'll clarify that in the commit message, thanks :+1:

File: addons/website/views/website_templates.xml [hunk 1]
```diff
@@ -243,6 +243,8 @@
             </t>
         </t>
         <link t-if="request and website" rel="canonical" t-att-href="website._get_canonical_url(canonical_params=canonical_params)"/>
+
+        <link rel="preconnect" href="https://fonts.gstatic.com/" crossorigin=""/>
```
Comment (mart-e) [inline addons/website/views/website_templates.xml]: Why not on fonts.odoocdn.com? Do we load google fonts by default?

Comment (qsm-odoo) [reply addons/website/views/website_templates.xml]: Yes, we load fonts from google directly for all website fonts (and for the v13.0, users can select any font from google fonts).

We could probably add a preconnect to odoocdn too for the Noto support font... not sure.

**Comment by qsm-odoo** [comment]
@robodoo r+

**Comment by qsm-odoo** [comment]
@robodoo wake up

**Comment by qsm-odoo** [comment]
cc @xmo-odoo ?

**Pull request closed without merging**